iset.mm

```diff
@@ -97573,6 +97724,256 @@ Knuth for iterated exponentiation (_Science_ 194, 1235-1242, 1976) and
+      enen2 bitrd cz c1 caddc co cmpt cfrec ensymd sizeennn eqeq12d cuz wf1 0zd
+      wf1o eqid frec2uzf1od f1of1 f1fveq syl12anc 3bitr4rd rexlimddv ) AFGZBFGZ
+      HZACIZJKZALMZBLMZNZABJKZTZCOVMVQCOPZVNVMWCCAUAUBUCVOVPOGZVQHZHZBDIZJKZWBD
+      OVNWHDOPZVMWEVNWIDBUAUBUDWFWGOGZWHHZHZVPWGJKZVPWGNZWAVTWLWDWJWMWNTVOWDVQW
+      KUEZWFWJWHUFZVPWGUGQWLWAVPBJKZWMWLVQWAWQTVOWDVQWKUHZAVPBUIRWLWHWQWMTWFWJW
+      HUJZBWGVPUKRULWLVTVPEUMEIUNUOUPUQSURZMZWGWTMZNZWNWLVRXAVSXBWLWDVPAJKVRXAN
+      WOWLAVPWRUSEAVPUTQWLWJWGBJKVSXBNWPWLBWGWSUSEBWGUTQVAWLOSVBMZWTVCZWDWJXCWN
+      TWLOXDWTVEXEWLESWTWLVDWTVFVGOXDWTVHRWOWPOXDVPWGWTVIVJULVKVLVL $.
+  $}
+
+  ${
+    $d A f $.  $d B f $.
+    $( The size of two finite sets is equal if and only if there is a bijection
+       mapping one of the sets onto the other.  (Contributed by Alexander van
+       der Vekens, 17-Dec-2017.) $)
+    sizeeqf1o $p |- ( ( A e. Fin /\ B e. Fin )
+        -> ( ( size ` A ) = ( size ` B ) <-> E. f f : A -1-1-onto-> B ) ) $=
+      ( cfn wcel wa csize cfv wceq cen wbr cv wf1o wex sizeen bren syl6bb ) ADE
+      BDEFAGHBGHIABJKABCLMCNABOABCPQ $.
+
+    $( There is no bijection between a finite set and an infinite set.  By
+       ~ infnfi the theorem would also hold if "infinite" were expressed as
+       ` _om ~<_ B ` .  (Contributed by Alexander van der Vekens,
+       25-Dec-2017.) $)
+    fiinfnf1o $p |- ( ( A e. Fin /\ -. B e. Fin )
+                      -> -. E. f f : A -1-1-onto-> B ) $=
+      ( cfn wcel cv wf1o wex f1ofi ex exlimdv con3dimp ) ADEZABCFZGZCHBDEZMOPCM
+      OPABNIJKL $.
+  $}
+
+  $( The codomain of an onto function is a set if its domain is a set.
+     (Contributed by AV, 4-May-2021.) $)
+  focdmex $p |- ( ( A e. V /\ F : A -onto-> B ) -> B e. _V ) $=
+    ( wcel wfo wa crn cvv wf fof anim2i ancomd rnexg 3syl wb forn eleq1d adantl
+    fex mpbid ) ADEZABCFZGZCHZIEZBIEZUDABCJZUBGCIEUFUDUBUHUCUHUBABCKLMABDCTCINO
+    UCUFUGPUBUCUEBIABCQRSUA $.
+
+  ${
+    $d A f $.  $d B f $.  $d F f $.
+    $( The size of two finite sets is equal if there is a bijection mapping one
+       of the sets onto the other.  (Contributed by Jim Kingdon,
+       21-Feb-2022.) $)
+    sizeeqf1oi $p |- ( ( A e. Fin /\ F : A -1-1-onto-> B )
+        -> ( size ` A ) = ( size ` B ) ) $=
+      ( vf cfn wcel wf1o wa csize cfv wceq cv wex f1odm eleq1d biimparc wfun wb
+      cdm mpbird f1ofun adantl fundmfibi syl simpr f1oeq1 spcegv sylc sizeeqf1o
+      f1ofi syldan ) AEFZABCGZHZAIJBIJKZABDLZGZDMZUNCEFZUMURUNUSCSZEFZUMVAULUMU
+      TAEABCNOPUNCQZUSVARUMVBULABCUAUBCUCUDTULUMUEUQUMDCEABUPCUFUGUHULUMBEFUOUR
+      RABCUJABDUIUKT $.
+  $}
+
+  ${
+    $d A f $.  $d B f $.  $d F f $.
+    $( The size of a finite set which is a one-to-one function is equal to the
+       size of the function's range.  (Contributed by Jim Kingdon,
+       21-Feb-2022.) $)
+    sizef1rn $p |- ( ( A e. Fin /\ F : A -1-1-> B )
+        -> ( size ` F ) = ( size ` ran F ) ) $=
```

Comment:
Is this result needed somewhere ? It is very encoding-dependent.

Reply:
Beats me. But if we don't like it we should remove it from set.mm too.

Reply:
Sorry, I don't know this idiom "beats me". Does it mean that you disagree ? What I mean is that the cardinality of size of a function is not a very natural notion (in set.mm, a function is encoded as a set of couples (argument, value), so it is coincidental that this is equal to the cardinality of the domain).  If you consider that a function is "a rule that associates with the elements of a set some elements of another set", then speaking of its size has no real meaning.
If you want to keep it, maybe add, as in set.mm, "(Avoid depending on this [encoding?] detail)"

Reply:
https://en.wiktionary.org/wiki/beats_me

Reply:
Thanks for the wiktionary link, Mario. And in this case it means "I don't know" or "I don't know enough to have an opinion" or something like that.

Reply:
So what about adding "(Avoid depending on this detail.)" ? (I checked it is the used formulation.)
For the present theorem, the important result is that for injections, size dom F = size ran F.